geomstats/geometry/riemannian_metric.py

```diff
@@ -290,6 +290,53 @@ def dist(self, point_a, point_b, **kwargs):
         dist = gs.sqrt(sq_dist)
         return dist
 
+    def dist_broadcast(self, point_a, point_b):
```

Comment:
Why not just including this in the `dist` function?

Reply:
To keep things simple: the output of `dist` is a scalar or 1d, where as `dist_broadcast` is always 2d. This would be more prone to errors